P/PETSc/build_tarballs.jl

```diff
@@ -26,12 +26,13 @@ else
   blas_64=0
 fi
 
-./configure CC=$CC FC=$FC CXX=$CXX --with-batch --PETSC_ARCH=$BUILD_AR  --with-blaslapack-lib=$BLAS_LAPACK_LIB --with-blaslapack-suffix=$BLAS_LAPACK_SUFFIX --known-64-bit-blas-indices=$blas_64 --with-mpi=0
+opt_flags="--with-debugging=0 COPTFLAGS='-O3 -march=native -mtune=native' -CXXOPTFLAGS='-O3 -march=native -mtune=native' FOPTFLAGS='-O3 -march=native -mtune=native"
+./configure $opt_flags CC=$CC FC=$FC CXX=$CXX --with-batch --PETSC_ARCH=$BUILD_AR  --with-blaslapack-lib=$BLAS_LAPACK_LIB --with-blaslapack-suffix=$BLAS_LAPACK_SUFFIX --known-64-bit-blas-indices=$blas_64 --with-mpi=0
```

Comment:
Quoting from https://www.mcs.anl.gov/petsc/documentation/installation.html

>  `PETSC_ARCH`: this variable gives a name to a configuration/build. Configure uses this value to stores the generated config makefiles in `${PETSC_DIR}/${PETSC_ARCH}`. Make uses this value to determine this location of these makefiles [which intern help in locating the correct include and library files]. 

So why is this set to `${BUILD_AR}` which holds the name of the [`ar`](https://en.wikipedia.org/wiki/Ar_(Unix)) command for the host?  Probably not a big deal, it just looks a bit odd

Reply:
I think I grabbed the wrong variable.